math/FunMath.h

```diff
@@ -4,6 +4,12 @@
 #include <vector>
 #include <string>
 
+/**
+ * Josiah Graham
+ * Faisal Lalani
+ * Kyle Hilton
+ */
+
```

Comment:
You don't need this comment for this exercise-but thanks for the thought!
(please remove it)